app/design/adminhtml/default/default/template/ebizmarts/mailchimp/system/config/form/field/array_dropdown.phtml

```diff
@@ -14,28 +14,28 @@ if (!$this->_addAfter) {
 $colspan = $colspan > 1 ? 'colspan="' . $colspan . '"' : '';
 ?>
 
-<div class="grid" id="grid<?php echo $this->quoteEscape($htmlId) ?>">
+<div class="grid" id="grid<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
```

Comment:
This can be more straight forward by calling only one function that calls the helper internally.